MJL20151028 - Created patch

This adds a heading to make it clear to readers that this is a mirror
of Eric Raymond's original site, and to offer a link back to my own
home page


--- index.html	2010-10-23 07:34:17.000000000 +1100
+++ index.html.mjl	2015-10-28 21:50:46.715806160 +1100
@@ -12,10 +12,19 @@
 <title>Jargon File Resources</title>
 </head>
 <body>
+
+<table width='100%' cellpadding='0' summary='Mirror page header'>
+<tr>
+<td width='30%'>Back to <a href='http://milosophical.me'>Mike's Home Page</a></td>
+<td width='40%'><em>This is a MIRROR of Eric Raymond's original site</em></td>
+<td width='30%' align='right'>28 Oct 2015</td>
+</table>
+<hr />
+
 <table width='100%' cellpadding='0' summary='Canned page header'>
 <tr>
-<td width='30%'>Back to <a href='http://catb.org/~esr/'>Eric's Home Page</a></td>
-<td width='30%' align='center'>Up to <a href='http://www.catb.org/~esr/sitemap.html'>Site Map</a></td>
+<td width='30%'>To <a href='http://catb.org/~esr/'>Eric's Home Page</a></td>
+<td width='30%' align='center'>Up to <a href='http://www.catb.org/~esr/sitemap.html'>CatB Site Map</a></td>
 <td width='30%' align='right'>01 Oct 2004</td>
 </table>
 <hr />
--- index-2.html	2010-10-23 07:34:17.000000000 +1100
+++ index-2.html.mjl	2015-10-28 21:50:58.591958675 +1100
@@ -12,10 +12,19 @@
 <title>Jargon File Resources</title>
 </head>
 <body>
+
+<table width='100%' cellpadding='0' summary='Mirror page header'>
+<tr>
+<td width='30%'>Back to <a href='http://milosophical.me'>Mike's Home Page</a></td>
+<td width='40%'><em>This is a MIRROR of Eric Raymond's original site</em></td>
+<td width='30%' align='right'>28 Oct 2015</td>
+</table>
+<hr />
+
 <table width='100%' cellpadding='0' summary='Canned page header'>
 <tr>
-<td width='30%'>Back to <a href='http://catb.org/~esr/'>Eric's Home Page</a></td>
-<td width='30%' align='center'>Up to <a href='http://www.catb.org/~esr/sitemap.html'>Site Map</a></td>
+<td width='30%'>To <a href='http://catb.org/~esr/'>Eric's Home Page</a></td>
+<td width='30%' align='center'>Up to <a href='http://www.catb.org/~esr/sitemap.html'> CatB Site Map</a></td>
 <td width='30%' align='right'>01 Oct 2004</td>
 </table>
 <hr />
--- news.html	2003-12-30 05:39:56.000000000 +1100
+++ news.html.mjl	2015-10-28 21:51:23.017272341 +1100
@@ -11,10 +11,19 @@
 <title>Jargon File News</title>
 </head>
 <body>
+
+<table width='100%' cellpadding='0' summary='Mirror page header'>
+<tr>
+<td width='30%'>To <a href='http://milosophical.me'>Mike's Home Page</a></td>
+<td width='40%'><em>This is a MIRROR of Eric Raymond's original site</em></td>
+<td width='30%' align='right'>28 Oct 2015</td>
+</table>
+<hr />
+
 <table width="100%" cellpadding="0" summary="Canned page header">
 <tr>
 <td width="30%">Back to <a href="index-2.html">Jargon File Resources</a></td>
-<td width="30%" align="center">Up to <a href="http://www.catb.org/~esr/sitemap.html">Site Map</a></td>
+<td width="30%" align="center">Up to <a href="http://www.catb.org/~esr/sitemap.html">CatB Site Map</a></td>
 <td width="30%" align="right">$Date: 2003/10/27 05:33:06 $</td>
 </tr>
 </table>
